selfdrive/car/toyota/carcontroller.py

```diff
@@ -114,7 +114,15 @@ def update(self, CC, CS, now_nanos):
       interceptor_gas_cmd = clip(pedal_command, 0., MAX_INTERCEPTOR_GAS)
     else:
       interceptor_gas_cmd = 0.
-    pcm_accel_cmd = clip(actuators.accel, self.params.ACCEL_MIN, self.params.ACCEL_MAX)
+
+    if CC.longActive:
+      accel_offset = CS.pcm_neutral_force / self.CP.mass
+    else:
+      accel_offset = 0.
```

Comment:
I don't know if we fully understand this signal yet, I just labeled my best guess looking at some data. Is there anything in techinfo, or the dealer software about this?

Reply:
only thing in the repair manual that I can find relating to feedback force (which I believe this is) of any kind is Cruise Request Driving Force Feedback Status, which I've also mentioned in my opendbc issue a few weeks ago (https://github.com/commaai/opendbc/issues/987), it has a few states but I don't exactly know which CAN ID this might be on, it might relate to how this is calculated in the stock system